js/damas.js

```diff
@@ -368,7 +371,7 @@
 				}
 			}
 		}
-		req.send("id="+id+"&keys="+Object.toJSON(keys));
+		req.send("id="+id+"&keys="+keys);
```

Comment:
I think that update_rest should process the keys to send in the same way than create_rest does